### PR TITLE
Use tox for multi version testing

### PR DESCRIPTION
**Pull request by FooBarQuaxx**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,5 +1,11 @@
 run-tests:
 	pipenv run pytest test_pipenv.py
+run-tox:
+	pipenv run tox
```
Comment (timofurrer) [inline Makefile]: I'm not sure if it's clever to use the module-under-test to actually run the tests. However, on the other side who is going to ensure that `tox` is actually installed ...

File: Makefile [hunk 2]
```diff
@@ -1,5 +1,11 @@
...
 init:
 	pip install pipenv
 	pipenv install --dev
+clean:
```
Comment (timofurrer) [inline Makefile]: What about `__pycache__`?

File: Makefile (hunk 1 above)
Comment (FooBarQuaxx) [inline Makefile]: The same thing is done on the previous makefile target (run-tests). I thought if it's good enough to run pytest, then it should be ok for running tox.

File: Makefile (hunk 2 above)
Comment (FooBarQuaxx) [inline Makefile]: I can get this part fixed easily.

**Comment by kennethreitz** [comment]
We're currently relying on travis-ci for this.

**Pull request closed without merging**